web/src/components/storage/utils.js

```diff
@@ -135,6 +135,20 @@ const deviceLabel = (device) => {
   return size ? `${name}, ${deviceSize(size)}` : name;
 };
 
+/**
+ * Checks if volume uses given fs. This method works same as in backend
+ * case insensitive.
+ *
+ * @param {import(~/clients/storage).Volume} volume
+ * @param {string} fs filesystem name to check
+ * @returns {boolean} true when volume uses given fs
```

Comment:
NP: if I'm not wrong, we're using dash and _title notation_. I.e., 

```suggestion
 * @param {string} fs - Filesystem name to check.
 * @returns {boolean} true when volume uses given fs
```